### DISCUSSION TITLE
iPadOS GitHub app can’t restore reading location

### DISCUSSION THREAD

**Discussion by minwang1** [community/community#127653] [Mobile]
I using this app reading markdown files,when I 
press home button navigate to launcher and wait for a wile,after that I switch back to this app,most of time page just scroll to the top rather than stay in the place at last time.I have test this in android app,it seems no happen.

Can you guys fix this?thx.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.164.0 (132484022)
Device: iPad (8th generation) (iOS 17.5.1)
</details>

**Comment by elisealix22** [comment]
Hi @minwang1 , thank you so much for reaching out and providing a detailed report! We are tracking this issue internally and I added your feedback to our open issue. We will look into it.
